### ISSUE TITLE
Implement the update todo functionality

### ISSUE THREAD

**Issue by harshalranjhani** [ACM-VIT/todo-app#4]
Add a button, to open a new form, which updates the todo title and category.

**Comment by Utkarsh212** [comment]
Please assign this issue to me

**Comment by harshalranjhani** [comment]
Assigned!, Thank you for showing interest.

**Comment by harshalranjhani** [comment]
Assigned! Thank you for showing interest.

**Comment by harshalranjhani** [comment]
> Please assign this issue to me
Assigned! Thank you for showing interest.

**Comment by nakshatraraghav** [comment]
@Utkarsh212 any updates, are you stuck anywhere?

**Comment by Utkarsh212** [comment]
Thanks for asking, @nakshatraraghav. I am working on it

**Comment by nakshatraraghav** [comment]
ok!

**Comment by harshalranjhani** [comment]
@Utkarsh212 what's the status?

**Comment by Utkarsh212** [comment]
@harshalranjhani After going through the source code, I discovered that the project required more react understanding than my current knowledge. Like implementation of state management using the useContext and useReducer Hooks etc. This channelled me into diving deep into learning more about these concepts. Considering this, I started exploring these concepts and now have the relevant knowledge to handle this issue.
Now, I will resolve this issue in a short time.
